ui/library/src/context/PageRenderContext.ts

```diff
@@ -0,0 +1,244 @@
+import * as React from 'react';
+import { pdfjs } from 'react-pdf';
+
+import { Nullable } from '../components/types/utils';
+import { logProviderWarning } from '../utils/provider';
+import { PageRotation } from '../utils/rotate';
+import { PageNumber } from './ScrollContext';
```

Comment:
Since this type is highly used, maybe placing it under `/types/page.ts` would be appropriate.

Reply:
@yensung done

Reply:
I am still seeing this type imported from ScrollContext

Reply:
@yensung i havent pushed the code yet xD My bad

Reply:
@yensung finally update